homeassistant/components/yale_smart_alarm/strings.json

```diff
@@ -2,7 +2,8 @@
   "config": {
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
-      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
+      "unique_id_mismatch": "The user identifier does not match the previous identifier"
```

Comment:
What I miss in this error is, what must the user do to fix this? Okay, my identifier mismatched, but users never had to do anything with an identifier